### DISCUSSION TITLE
Fleet final pipeline errors

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#12814] [2.4] [answered]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

64

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello!

2.4.60, in logs from Win agents we get "error.message" with values like "failed in Fleet agent final_pipeline: Text '2024-04-17T13:14:03.808Z' could not be parsed at index 20".
We checked ".fleet_final_pipeline-1" and it looks like error in 
```
  {
    "date": {
      "if": "ctx.event?.module == 'system'",
      "field": "event.created",
      "target_field": "@timestamp",
      "formats": [
        "yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'"
      ]
    }
  }
```
because correct timestamp is "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'".
This pipeline is managed one, so question is: can we fix it by editing pipeline or we should wait for fix?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
Can you provide any screenshots for what you're seeing in SOC?

**Reply by Sahale** [reply]
I'm sorry, but I didn't get what specific screenshots you need.
Anyway, screenshot from Elastic and SOC Grid.

Elastic:
![telegram-cloud-photo-size-2-5289751141766256710-y](https://github.com/Security-Onion-Solutions/securityonion/assets/10064214/b63b8f8c-2ce6-4bb7-8fed-8d4858d37085)
Node status:
<img width="362" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/10064214/e4c35009-92d1-42a2-b445-46d1c713e828">

We know about high IOwait on this node, but we got same errors from nodes with normal IOwait.

**Reply by reyesj2** [reply, marked as answer]
That error will be fixed with https://github.com/Security-Onion-Solutions/securityonion/pull/12881
